src/components/vue-form-renderer.vue

```diff
@@ -88,6 +99,24 @@ export default {
       scrollable: null,
     };
   },
+  computed: {
+    containerClass() {
+      return this.parentScreen ? `screen-${this.parentScreen}` : 'custom-css-scope';
+    },
+    cssDevice() {
+      const { devicePixelRatio } = window;
+      const deviceDensity = devicePixelRatio === undefined ? 1 : devicePixelRatio;
```

Comment:
do we care about the device pixel ratio? Seems like we just care about the device width?

Reply:
You are right. It will be removed